bazel/linux/templates/runner.template.sh

```diff
@@ -22,39 +22,39 @@ function help {
 This script executes your target in an emulator.
 Prefer to use as:
 
-  bazel run $TARGET -- [-k kernel option]... [-e emu option]... [-r rootfs] [-s|-x|-h]
+  bazel run $TARGET -- [--runner-kernel-opts kernel option]... [--runner-emulator-opts emu option]... [--runner-rootfs rootfs] [--runner-single|--runner-show-state|--runner-help]
```

Comment:
you also need to update the message in the `onexit` function, and all the Examples provided below.

Alternatives:
1) Leave the options as they are, support `--` like common? so eg, `bazel run :target -- -runner-opt -- -script-opt`
2) Like we have `-k` kernel option, `-e` emulator option, have a `-a` script in VM argument?